chemprop/v2/cli/convert.py

```diff
@@ -0,0 +1,44 @@
+import sys
+import torch
+import logging
+from pathlib import Path
+from argparse import ArgumentParser, Namespace
+
+from chemprop.v2.cli.utils import Subcommand
+from chemprop.v2.utils.v1_to_v2 import convert_model_file_v1_to_v2
+
+logger = logging.getLogger(__name__)
+
+class ConvertSubcommand(Subcommand):
+    COMMAND = "convert"
+    HELP = "convert a v1 model checkpoint (.pt) to a v2 model checkpoint (.ckpt)"
+
+    @classmethod
+    def add_args(cls, parser: ArgumentParser) -> ArgumentParser:
+        parser.add_argument(
+            "input_path",
+            help="Path to a v1 model .pt checkpoint file.",
+        )
+        parser.add_argument(
+            "output_path",
+            nargs="?",
+            help="Path at which the converted model will be saved. If not specified, it defaults to the name of the input file suffixed with '_v2.ckpt' in your current working directory.",
+        )
+        return parser
+
+    @classmethod
+    def func(cls, args: Namespace):
+        args.input_path = Path(args.input_path)
+        args.output_path = Path(args.output_path or args.input_path.parent / (args.input_path.stem + "_v2.ckpt"))
+
+        logger.info(f"Converting v1 model checkpoint '{args.input_path}' to v2 model checkpoint '{args.output_path}'...")
+        convert_model_file_v1_to_v2(str(args.input_path), str(args.output_path))
```

Comment:
`args.input_path` starts as a string. You then convert it to a path using `Path()`. The on line 35 you convert it back to a string. Would it be okay to just leave `args.input_path` and `args,output_path` as strings?

Reply:
can the converter function not accept `Path`s?

Reply:
I thought it couldn't, but I just tested and it seemed to work fine with `Paths`, so I changed this part of code to pass just `args.input_path` and `args.output_path` in without converting them into `str`.